string instructionID 
float32 followerA_x
float32 followerA_y
float32 followerB_x
float32 followerB_y         
float32 followerC_x
float32 followerC_y